lib/core.mli

```diff
@@ -409,26 +434,34 @@ val witness : t -> string
 type substrings = Group.t
 (** Alias for {!Group.t}. Deprecated *)
 
+[@@ocaml.deprecated "Use Group.get"]
 val get : Group.t -> int -> string
 (** Same as {!Group.get}. Deprecated *)
```

Comment:
Those annotations should be below the declaration in OCaml (you are confused with Reason, I guess).